exercises/gigasecond/README.md

```diff
@@ -4,7 +4,7 @@ Calculate the moment when someone has lived for 10^9 seconds.
 
 A gigasecond is 10^9 (1,000,000,000) seconds.
 
-If you're unsure what operations you can perform on `DateTime<Utc>` take a look at the [chrono crate](https://docs.rs/chrono/0.4.0/chrono/) which is listed as a dependency in the `Cargo.toml` file for this exercise.
+If you're unsure what operations you can perform on `DateTime<Utc>` take a look at the [chrono crate](https://crates.io/crates/chrono) which is listed as a dependency in the `Cargo.toml` file for this exercise.
```

Comment:
I think that it's better to point directly at the documentation than to crates.io. The issue was that we'd previously been pointing to a version of the documentation that has since become outdated.

The suggested link will automatically redirect to the most current version of the documentation, which should keep links fresh. 
```suggestion
If you're unsure what operations you can perform on `DateTime<Utc>` take a look at the [chrono crate](https://docs.rs/chrono) which is listed as a dependency in the `Cargo.toml` file for this exercise.
```